.circleci/config.yml

```diff
@@ -39,30 +39,42 @@ jobs:
             docker exec cimg_ruby bash -c './test/bin/setup_tinytds_db.sh'
 
       - run:
-          name: compile openssl library
+          name: bundle install gems
           command: |
-            docker exec cimg_ruby bash -c 'sudo -E ./test/bin/install-openssl.sh'
+            docker exec cimg_ruby bash -c 'bundle install'
 
       - run:
-          name: compile freetds library
+          name: Write used versions into file
           command: |
-            docker exec cimg_ruby bash -c 'sudo -E ./test/bin/install-freetds.sh'
+            docker exec cimg_ruby bash -c 'bundle exec rake ports:version_file'
+
+      - restore_cache:
+          name: restore ports cache
+          keys:
+            - ports-<< parameters.ruby_version >>-{{ checksum ".ports_versions" }}
+            - ports-<< parameters.ruby_version >>-
 
       - run:
-          name: bundle install gems
+          name: compile ports and build gem
           command: |
-            docker exec cimg_ruby bash -c 'bundle install'
+            docker exec cimg_ruby bash -c 'bundle exec rake ports && bundle exec rake build'
```

Comment:
I recommend keeping "rake ports" and "rake build" as separate CI steps unless they absolutely need to be combined

Reply:
addressed in [69a6dbd](https://github.com/rails-sqlserver/tiny_tds/pull/533/commits/69a6dbd0f747bf193dbf95085ee0b984aae24af1).